### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by avr757**
This updates fixed the old links regarding the new API key changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,7 +69,7 @@ Esri welcomes contributions from anyone and everyone. Please see our [guidelines
 
 ## Licensing
 
-Copyright 2022 Esri
+Copyright 2024 Esri
```
Comment (mmgaw113) [inline README.md:72]: I don't know what the rules are for updating copyright. Did Andres say we should be updating the copyright for samples?

**Review by mmgaw113 — APPROVED** [review]
Just address the Copyright comment if needed, otherwise looks good.

File: README.md (hunk 1 above)
Comment (ZackAllen) [reply README.md:72]: Copyright is set to current year on file creation and file rename. Copyright is not changed on modification.

Comment (avr757) [reply README.md]: So the PM recommends a year range (2022-2024) for this copyright in the readme file as repo is having ongoing updates

**Pull request merged**